kafka/config_test.go

```diff
@@ -124,3 +124,29 @@ func TestConfigMapAPIs(t *testing.T) {
 	}
 
 }
+
+// Test that plugins will always be configured before their config options
+func TestConfigPluginPaths(t *testing.T) {
+	config := &ConfigMap{
+		"plugin.library.paths": "monitoring-interceptor",
+	}
+	_, err := config.convert()
+	if err != nil {
+		t.Skipf("Missing monitoring-interceptor.so, %s", err)
```

Comment:
nit: drop .so suffix since many developers are on osx and they might get confused